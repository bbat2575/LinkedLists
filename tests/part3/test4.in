NEW 0
VIEW-NESTED 0
NEW 1
1
NEW 2
1
2.2
NEW 3
@
one
two
VIEW-NESTED 3
NEW 3
{0}
1
2
VIEW-NESTED 4
INSERT 0 0 that's what she said!
VIEW-NESTED 4
INSERT 0 -1 q-_-p
VIEW-NESTED 4
NEW 3
{0}
1
{2}
VIEW-NESTED 5
DELETE 2 -1
VIEW-NESTED 5
NEW 3
{0}
{1}
{2}
VIEW-NESTED 6
INSERT 1 1 22e-1
VIEW-NESTED 6
NEW 6
{0}
2e6
{1}
hmm
{2}
#
VIEW-NESTED 7
VIEW ALL
