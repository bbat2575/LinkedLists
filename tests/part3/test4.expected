
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 

> 
> List 1: 1

> List 2: 1 -> 2.20

> List 3: @ -> one -> two

> @ -> one -> two

> Nested 4: {List 0} -> 1 -> 2

> {} -> 1 -> 2

> List 0: that's what she said!

> {that's what she said!} -> 1 -> 2

> List 0: that's what she said! -> q-_-p

> {that's what she said! -> q-_-p} -> 1 -> 2

> Nested 5: {List 0} -> 1 -> {List 2}

> {that's what she said! -> q-_-p} -> 1 -> {1 -> 2.20}

> List 2: 1

> {that's what she said! -> q-_-p} -> 1 -> {1}

> Nested 6: {List 0} -> {List 1} -> {List 2}

> {that's what she said! -> q-_-p} -> {1} -> {1}

> List 1: 1 -> 2.20

> {that's what she said! -> q-_-p} -> {1 -> 2.20} -> {1}

> Nested 7: {List 0} -> 2000000.00 -> {List 1} -> hmm -> {List 2} -> #

> {that's what she said! -> q-_-p} -> 2000000.00 -> {1 -> 2.20} -> hmm -> {1} -> #

> Number of lists: 8
List 0
List 1
List 2
List 3
Nested 4
Nested 5
Nested 6
Nested 7

> 
